app/med-list-slider/med-list-slider.component.html

```diff
@@ -0,0 +1,8 @@
+<ion-item-sliding *ngIf="doseMedication">
```

Comment:
I don't see where you have options for sliding.  Recommend if you are not using the sliding functionality to change it to a regular list.